recipes/cpplogging/all/conanfile.py

```diff
@@ -0,0 +1,100 @@
+from conans import CMake
+from conan import ConanFile
+from conan.tools import scm, build, files
+from conan.errors import ConanInvalidConfiguration
+import os
+import glob
+
+
+class CpploggingConan(ConanFile):
+    name = "cpplogging"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/chronoxor/CppLogging"
+    description = "C++ Logging Library provides functionality to log different events with a high" \
+        " throughput in multithreaded environment into different sinks (console, files, rolling files," \
+        " syslog, etc.). Logging configuration is very flexible and gives functionality to build flexible"\
+        " logger hierarchy with combination of logging processors (sync, async), filters, layouts (binary,"\
+        " hash, text) and appenders."
+    topics = ("performance", "logging", "speed", "logging-library", "low-latency")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"fPIC": [True, False],
+               "shared": [True, False]}
+    default_options = {"fPIC": True,
+                       "shared": False}
+    generators = "cmake", "cmake_find_package"
+    exports_sources = ["patches/**", "CMakeLists.txt"]
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def _configure_cmake(self):
+        if not self._cmake:
+            self._cmake = CMake(self)
+            self._cmake.definitions["CPPLOGGING_MODULE"] = "OFF"
+            self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "9",
+            "Visual Studio": "15",
+            "clang": "5",
+            "apple-clang": "10",
+        }
+
+    def requirements(self):
+        self.requires("zlib/1.2.13")
+        if self.version < "1.0.3.0" :
+            self.requires("cppcommon/1.0.2.0")
+        else:
+            self.requires("cppcommon/1.0.3.0")
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+        if self.settings.compiler.get_safe("cppstd"):
+            build.check_min_cppstd(self, "17")
```

Comment:
Looking over the project is does not say this and the GitHub actions build without it so I am surprsed to see this here 🤔 

Can you share why you are adding this?

Reply:
The project uses std::string_view which was added to the stl in C++17.

Reply:
fair enough, but it should be pointed by the upstream, there is no reference about C++ standard there.